.eslintrc

```diff
@@ -1,3 +1,6 @@
 {
     "extends": "girder",
+    "rules": {
+        "no-throw-literal": "off"
```

Comment:
Throwing an error in a promise will cause it to be rejected.  This is actually one of the incompatible changes in jquery 3 to conform to the A+ specification.  I think @brianhelba is throwing `undefined` there to be compatible with the previous behavior (the promise was rejected with no argument); however, I don't think there is any real chance anyone would rely on that behavior.  If that is indeed the motivation, I would prefer to just throw an actual error instead.